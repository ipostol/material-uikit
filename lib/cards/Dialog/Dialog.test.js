'use strict';

var _react = require('react');

var _react2 = _interopRequireDefault(_react);

var _enzyme = require('enzyme');

var _Dialog = require('./Dialog');

var _Dialog2 = _interopRequireDefault(_Dialog);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

describe('Dialog component', function () {

  it('should render correctly', function () {

    var props = {};

    var wrapper = (0, _enzyme.shallow)(_react2.default.createElement(_Dialog2.default, props));

    expect(wrapper).toMatchSnapshot();
  });
});