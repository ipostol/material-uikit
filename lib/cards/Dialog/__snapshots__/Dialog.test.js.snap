// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog component should render correctly 1`] = `
<div
  className="general"
>
  <div
    className="header"
  />
  <div
    className="description"
  />
  <div
    className="wrap"
  >
    <Button
      className="button"
      element="button"
      mod="tertiary"
      round="small"
    >
      Cancel
    </Button>
    <Button
      className="button"
      element="button"
      mod="secondary"
      round="small"
    >
      Delete
    </Button>
  </div>
  <Button
    className="close"
    element="button"
    mod="icon"
    round="small"
  >
    <Close />
  </Button>
</div>
`;
